src/less/chat.less

```diff
@@ -23,6 +24,118 @@
   }
 }
 
+.chat-card-custom {
+  .chat-card-custom-item{
+    img{
+      margin:5px;float:left;
+      border:none;
+      border-radius: 10px;
+      width:30px;
```

Comment:
`css` properties should have proper white spacing applied. The preferred method would be `property: value;`, so that there is a space between the colon and the value.

Reply:
done